docs/Limitations-and-Common-Issues.md

```diff
@@ -1,19 +1,35 @@
-# Limitations and Common Issues 
+# Frequently Asked Questions
```

Comment:
FAQ and limitations could probably be separate pages.

Reply:
Will change.